now.json

```diff
@@ -4,5 +4,11 @@
   "env": {
     "GH_TOKEN": "@gh-token-public"
   },
-  "version": 2
+  "version": 2,
+  "routes": [
+    {
+      "src": "/kernels/(.*)",
```

Comment:
You could create a dynamic route in Next.js to catch these: https://nextjs.org/docs#dynamic-routing